ipn/ipnlocal/local.go

```diff
@@ -6384,9 +6384,10 @@ func (b *LocalBackend) SuggestExitNode() (response apitype.ExitNodeSuggestionRes
 	return res, err
 }
 
-// suggestLastExitNode formats a response with the last suggested exit node's ID and name.
+// formatLastSuggestedExitNode formats a response with the last suggested exit node's ID and name.
+// Returns error if there is no id or name.
 // Used as a fallback before returning a nil response and error.
-func suggestLastExitNode(lastSuggestedExitNode lastSuggestedExitNode) (res apitype.ExitNodeSuggestionResponse, err error) {
+func formatLastSuggestedExitNode(lastSuggestedExitNode lastSuggestedExitNode) (res apitype.ExitNodeSuggestionResponse, err error) {
```

Comment:
oh! this isn't using the field on LocalBackend. I misread this earlier.

For very short scopes (like 3 lines) you can use a short variable name like "last" or "n".

Neither "suggest" or "format" really describe what this is doing... it's just converting one type to another type.

Maybe it can be a method on lastSuggestedExitNode? Like

```go
func (n lastSuggestedExitNode) asAPIType() (res apitype.ExitNodeSuggestionResponse, _ error) {
```
?

I guess the only reason you don't use `apitype.ExitNodeSuggestionResponse` directly and get rid of `lastSuggestedExitNode` is because you don't need the location field?